string_manipulation/helper.py

```diff
@@ -0,0 +1,53 @@
+
+
+def remove_multy_occurence(multy_occuring_string):
+    string_after_single_occurence = ""
+
+    for index_for_multy_occurence in multy_occuring_string:
+        if index_for_multy_occurence.lower() not in string_after_single_occurence.lower():
+            string_after_single_occurence = string_after_single_occurence + index_for_multy_occurence
+
+    return string_after_single_occurence
+
+
+def repeats(repeating_character_string, key_character):
+    counter = 0
+    for index_for_repeating_character in repeating_character_string:
+        if index_for_repeating_character.lower() == key_character.lower():
+            counter = counter + 1
+
+    if counter>1:
+        return True
+    else:
+        return False
+
+
+def remove_given_character(removing_string, key_character):
```

Comment:
Do you think `input_string` is a better name than `removing_string`?
Replace if yes.